src/pages/index.tsx

```diff
@@ -201,66 +73,22 @@ const AdminPage: NextComponentType = () => {
         <div className="col-12 col-md-6">
           <h1>Hello, Admin 👋</h1>
         </div>
-        <div className="col-12 col-md-6">
-          <form className="form-inline float-md-right" onSubmit={formik.handleSubmit}>
-            <div className="form-group ml-sm-3 mr-3 mb-2">
-              <label htmlFor="adminName" className="mr-3">
-                Your Name
-              </label>
-              <input
-                type="text"
-                className="form-control"
-                id="adminName"
-                placeholder="Your name"
-                onChange={formik.handleChange}
-                onBlur={formik.handleBlur}
-                value={formik.values.adminName}
-              />
-            </div>
-            <button type="submit" className="btn btn-primary mb-2">
-              Save
-            </button>
-          </form>
-        </div>
       </div>
       <div className="row">
         <div className="col-12 col-md-6">
           <div className="card">
             <div className="card-body">
-              <h2 className="font-weight-normal">Support Queue</h2>
-              <button className="btn btn-dark w-100 mt-4" onClick={getNext} disabled={counts.ideaCount + counts.technicalCount === 0}>
-                Get Next in Queue ({counts.ideaCount + counts.technicalCount})
-              </button>
-              {lastPop && (
-                <div className="alert alert-info mt-3">
-                  {
-                  (lastPop.supportRequest !== null) ?  
-                    (lastPop.userNotified
-                      ? `${lastPop.supportRequest.name} has been notified to come over`
-                      : `There was a problem notifying ${lastPop.supportRequest.name}, send them a message and tell them you're ready for them`)
-                    : 'It appears you are trying to access a support request that does not exist!'}
+              <h2 className="font-weight-normal">Config Items</h2>
+              {results.length === 0 && <div className="alert alert-info mt-3">No config items to display 🤔</div>}
+
+              {configItems.map((configItem) => (
+                <div className="form-group ml-sm-3 mr-3 mb-2">
```

Comment:
Missing the key on here, which was probably what the rule was complaining about. A key is just a value that is unique from the other items in the loop so React can know which is which when it's rendering an update. It's also best to not make it the index, because otherwise if the order changes it won't know if the order changed or if the contents of the item changed.
```suggestion
                <div key={configItem.key} className="form-group ml-sm-3 mr-3 mb-2">
```